### PR TITLE
feat(sdk): add custom bridge address for ECO

### PR DESCRIPTION
**Pull request by jakim929**
Adds the ECO bridge to the sdk

https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/366/files

- The bridge has slightly different interface so requires a custom `supportsTokenPair`. Created a new eco-bridge adapter.
- adds ECO bridge to constants
- adds viem and some helper functions

### CODE REVIEW

**Comment by jakim929** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5695** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5695" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5695?utm_source=stack-comment).

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Can we add a vitest test?   I can help pair on it

File: packages/sdk/test-next/testUtils/viemClients.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import {
+  createTestClient,
+  createPublicClient,
+  createWalletClient,
+  http,
+} from 'viem'
+import { goerli, optimismGoerli } from 'viem/chains'
```
Comment (roninjin10) [inline packages/sdk/test-next/testUtils/viemClients.ts:7]: I might make this it's own package later

File: packages/sdk/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "lint:fix": "yarn lint:check --fix",
     "pre-commit": "lint-staged",
     "test": "hardhat test",
-    "test:next": "vitest test-next/proveMessage.spec.ts",
+    "test:next": "vitest run",
```
Comment (roninjin10) [inline packages/sdk/package.json:20]:
```suggestion
    "test:next": "vitest",
    "test:next:run": "vitest run",
```

**Review by roninjin10 — APPROVED** [review]

**Review by twoshark — APPROVED** [review]

**Pull request merged**